apollo-backend/src/main/java/io/logz/apollo/configuration/DeploymentConfiguration.java

```diff
@@ -0,0 +1,21 @@
+package io.logz.apollo.configuration;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class DeploymentConfiguration {
+    private int forceCancelTimeoutSeconds;
+
+    @JsonCreator
+    public DeploymentConfiguration(@JsonProperty("timeoutInSeconds") int forceCancelTimeoutSeconds) {
+        this.forceCancelTimeoutSeconds = forceCancelTimeoutSeconds;
```

Comment:
it's not critical, but nonetheless
we are allowing to set a negative timeout
should we add validations for such cases?

Reply:
you are correct but all apollo is like this, I can simply add a check :)

Reply:
if it's by design, then no problem